test/auto_schedule/bilateral.cpp

```diff
@@ -0,0 +1,152 @@
+#include "Halide.h"
```

Comment:
Seems like the autoscheduler test should be integrated into apps/bilateral_grid, rather than duplicating most of the code here.